### PR TITLE
added parsing csv

### PR DESCRIPTION
**Pull request by TatianaPirch**
1. added dependencies for lombok and log4j
	2. added log4j.properties
	3. added entity (for testing)
	4. added ReviewRepository (for testing)
	5. added ServiceParsing
	6. added DataInsertController
	7. added log_file.log

### CODE REVIEW

File: log/log_file.log [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+
+2019-11-06 14:25:18 INFO  ServiceParsing:69 - Read number of items: 100000
```
Comment (boroda4436) [inline log/log_file.log]: @TatianaPirch should we push it?

File: src/main/java/team/five/amazonreviewutil/controller/DataInsertController.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import java.io.IOException;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import team.five.amazonreviewutil.service.ServiceParsing;
+
+@Component
+public class DataInsertController {
+
+    private static final String DATA_FILE_NAME = "D:\\My\\MA\\project\\Reviews.csv";
```
Comment (boroda4436) [inline src/main/java/team/five/amazonreviewutil/controller/DataInsertController.java]: @TatianaPirch what if I will run this on my Mac? Will it works?

File: src/main/java/team/five/amazonreviewutil/repositiry/ReviewRepository.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
+package team.five.amazonreviewutil.repositiry;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import team.five.amazonreviewutil.entity.Review;
+
+public interface ReviewRepository extends JpaRepository<Review, Long> {
+}
```
Comment (boroda4436) [inline src/main/java/team/five/amazonreviewutil/repositiry/ReviewRepository.java]:
```suggestion
}

```

File: src/main/java/team/five/amazonreviewutil/service/ServiceParsing.java [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+    private static Logger logger = Logger.getLogger(ServiceParsing.class);
+
+    private final int fieldsCount = 10;
+    private int itemsRead;
+    private final String fieldSeparator = ",";
+    private final String charsetName = "UTF-8";
+    Charset charset = Charset.forName(charsetName);
+    private final char lineBreak = '\n';
+    private static final String DATA_FILE_NAME = "D:\\My\\MA\\project\\test2.csv";
```
Comment (boroda4436) [inline src/main/java/team/five/amazonreviewutil/service/ServiceParsing.java]: @NikolaiKatsai  @SkaterWeather  can you run this on your PC? Is everything ok?

File: src/main/java/team/five/amazonreviewutil/service/ServiceParsing.java [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+    @Autowired
+    ReviewRepository reviewRepository;
+
+    private static Logger logger = Logger.getLogger(ServiceParsing.class);
+
+    private final int fieldsCount = 10;
+    private int itemsRead;
+    private final String fieldSeparator = ",";
+    private final String charsetName = "UTF-8";
+    Charset charset = Charset.forName(charsetName);
```
Comment (boroda4436) [inline src/main/java/team/five/amazonreviewutil/service/ServiceParsing.java]:
```suggestion
    Charset charset = StandardCharsets.UTF_8;
```

File: src/main/java/team/five/amazonreviewutil/service/ServiceParsing.java [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+    private final char lineBreak = '\n';
+    private static final String DATA_FILE_NAME = "D:\\My\\MA\\project\\test2.csv";
+
+    /*public static void main(String[] args) throws IOException {
+        ServiceParsing serviceParsing = new ServiceParsing();
+        serviceParsing.processInputFile(DATA_FILE_NAME);
+    }*/
+
+    public void processInputFile(String dataFileName) throws IOException {
```
Comment (boroda4436) [inline src/main/java/team/five/amazonreviewutil/service/ServiceParsing.java]: @TatianaPirch How long it takes to process the file?

**Review by boroda4436 — CHANGES_REQUESTED** [review]

File: log/log_file.log (hunk 1 above)
Comment (TatianaPirch) [reply log/log_file.log]: I attached a log file to show how long it took to process csv-file

File: src/main/java/team/five/amazonreviewutil/service/ServiceParsing.java (hunk 6 above)
Comment (TatianaPirch) [reply src/main/java/team/five/amazonreviewutil/service/ServiceParsing.java]: about 2 minutes

File: src/main/java/team/five/amazonreviewutil/controller/DataInsertController.java (hunk 2 above)
Comment (SkaterWeather) [reply src/main/java/team/five/amazonreviewutil/controller/DataInsertController.java]: It's will not on work in another Windows(like mine or Kolya's) too;
Idk, i tried to see reliable path file in idea, but it don't worked for me;
@TatianaPirch please place .csv file in package src/main/resources/ and push it in branch too;

Comment (SkaterWeather) [reply src/main/java/team/five/amazonreviewutil/controller/DataInsertController.java]: String txt = "./src/resources/Reviews.csv" , try it out, maybe will work for you

File: src/main/java/team/five/amazonreviewutil/service/ServiceParsing.java (hunk 4 above)
Comment (SkaterWeather) [reply src/main/java/team/five/amazonreviewutil/service/ServiceParsing.java]: Same, i responsed

File: src/main/java/team/five/amazonreviewutil/service/CsvParserService.java [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+    private static Logger logger = Logger.getLogger(CsvParserService.class);
+
+    private final int fieldsCount = 10;
+    private int itemsRead;
+    private final String fieldSeparator = ",";
+    Charset charset = StandardCharsets.UTF_8;
+    private final char lineBreak = '\n';
+
+    public void processInputFile(String fileName) throws IOException {
```
Comment (boroda4436) [inline src/main/java/team/five/amazonreviewutil/service/CsvParserService.java]: @TatianaPirch Isn't it too much for one method??? Did you consider to separate the functionality?

**Review by boroda4436 — CHANGES_REQUESTED** [review]

**Pull request merged**